### PR TITLE
Fixed hamburger menu ,colour, search icon, greeting phrase

### LINKED ISSUE #9: Making Hamburger function Properly
# ✨ Feature Request

### **Description of the Feature**

Making Hamburger function Properly 

image

fixing top and bottom 

### **Alternatives Considered**

Points will be assigned according to PR content 

### **Are you participating in GSSoC?**
- [X] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by 1-SubhamSingh**
## Related Issue
Issue : fix the hamburger menu

Fixes : #9 

https://github.com/user-attachments/assets/92324e28-4263-4ffe-95cc-ab1613950b66

## Description of Changes
Changed the colour of the menu button , enhanced the animation of the menu button.
Fixed the bottom of the hamburger menu , adjusted its size by blocking its overflow.
Positioned the greeting phrase in the center to look good.
Moved the search button to the right with some padding which looks good on different screen sizes.
No fixed the top because the search icon would have hidden due to that.

https://github.com/user-attachments/assets/35b7ea32-c05c-4464-aba4-4506e614cb21

## Type of Pull Request
- [* ] Performance Improvement

## Checklist for Contributor
use'X' in [ ] to check it .

- [* ] I have reviewed my code before submission.
- [* ] I have read and followed the project's Contribution Guidelines.
- [* ] I have tested the code thoroughly.
- [* ] I have attached related issue numbers, screenshots, or videos (if applicable).
- [* ] I have checked the existing code base and my changes do not introduce any breaking changes.
- [ *] My PR addresses only one issue; I have created separate PRs for other issues.
- [* ] I have ensured that the PR does not include unrelated files.
- [* ] I am mindful of the project's coding standards and best practices.

### CODE REVIEW

File: frontend/src/Pages/Herosection.css [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-   background-image: url('../assets/mixbg.png');
+    background-image: url('../assets/mixbg.png');
     background-size: cover;
     background-position: center;
     height: 100vh;
-    width: 100%;
-}
-
+    width: 100vw; 
+    overflow: hidden;
```
Comment (dhairyagothi) [inline frontend/src/Pages/Herosection.css]: Please remove it 
Do  `overflow-x : hidden`

**Review by dhairyagothi — COMMENTED** [review]
Just do this change other thing is perfect

File: frontend/src/Pages/Herosection.css (hunk 1 above)
Comment (1-SubhamSingh) [reply frontend/src/Pages/Herosection.css]: ok

**Comment by 1-SubhamSingh** [comment]
Sorry forget to remove the previous overflow line , wait a minute

**Comment by dhairyagothi** [comment]
Perfect to go now @1-SubhamSingh good job

**Comment by 1-SubhamSingh** [comment]
Thanks for the PR and also for updating the label to level 2 :))

**Pull request merged**